src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -65,6 +65,8 @@ public static class ScriptConstants
         public const string FunctionMetadataFileName = "function.json";
         public const string ProxyMetadataFileName = "proxies.json";
         public const string ExtensionsMetadataFileName = "extensions.json";
+        public const string AppOfflineFileName = "app_offline.htm";
+        public const string ResourcePath = "Microsoft.Azure.WebJobs.Script.WebHost.Resources";
```

Comment:
Not expecting that to happen, but assembly name changes would impact this, leaving it out of sync. Would a simple test to validate the path be worth it?

Reply:
The tests I've added will fail if this path is wrong